mipidsi/src/batch.rs

```diff
@@ -1,7 +1,7 @@
 //! Original code from: [this repo](https://github.com/lupyuen/piet-embedded/blob/master/piet-embedded-graphics/src/batch.rs)
 //! Batch the pixels to be rendered into Pixel Rows and Pixel Blocks (contiguous Pixel Rows).
 //! This enables the pixels to be rendered efficiently as Pixel Blocks, which may be transmitted in a single Non-Blocking SPI request.
-use crate::{models::Model, Display, Error};
+use crate::{error::Error, models::Model, Display};
```

Comment:
At some point we might want to reformat all imports to be more consistent across the crate, but IIRC most rustfmt options for that aren't stabilized yet. If there was a fixed order of internal vs. external imports and the same grouping policy for all imports the diffs would be easier to read. But that's all outside the scope of this PR.